.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -14,6 +14,10 @@ A clear and concise description of what the bug is.
 **Examples**
 If applicable, add screenshots to help explain your problem.
 
+**Reproductions**
+If applicable, provide a reproduction on platforms like [stackblitz](https://stackblitz.com)
```

Comment:
I'd propose we use runkit as an example, it's already supported on NPM; than promoting a commercial product. See https://npm.runkit.com/validator

Reply:
seems good